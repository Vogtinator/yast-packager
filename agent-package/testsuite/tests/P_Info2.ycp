{
    return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"german","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Read(.longDesc,"alien");
}
{
    return SCR::Read(.delDesc,"alien");
}
{
    return SCR::Read(.delDesc,"aaa_base");
}
{
    return SCR::Read(.notifyDesc,"alien");
}
