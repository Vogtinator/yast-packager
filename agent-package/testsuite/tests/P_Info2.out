(true)
("Experimentelles Perl-Skript, um Softwarepakete zu konvertieren.  Unterstützt werden die Paketformate deb (Debian), rpm (Red Hat) und tgz (Slackware). \n")
("")
("Dieses Paket ist zum Betrieb bzw. für die Verwaltung des Linux-Systems zwingend erforderlich! \nDeinstallieren Sie es bitte nur, wenn Sie sicher sind, dass Sie genau wissen was Sie tun! \n")
("")
